test/integration/secrets_test.rb

```diff
@@ -0,0 +1,44 @@
+require "test_helper"
+
+class SecretsTest < ActionDispatch::IntegrationTest
+  test "#create unauthorized" do
+    post secrets_path
+    assert_response :unauthorized
+  end
+
+  test "#create with faulty token (encoded with different signing key)" do
+    post secrets_path, headers: { "Authorization" => "Bearer #{jwt_unauthorized}" }
+    assert_response :unauthorized
+  end
+
+  test "#create or update a secret" do
```

Comment:
Did you mean to assert something here?

Reply:
well the "create_secret" call is doing the test, but i then need to reuse that for show/delete

Reply:
i can move the assertions from there to here